docs/pages/blog/mui-x-mid-v6-features.md

```diff
@@ -0,0 +1,129 @@
+---
+title: MUI X improvements, Charts in Alpha, Filtering, and more!
+description: Support for time zones, a new filtering UI, charts and more..
+date: 2023-08-07T00:00:00.000Z
+authors: ['richbustos', 'josefreitas']
+card: true
+tags: ['MUI X', 'News']
+---
+
+<img src="/static/blog/mui-x-mid-v6-features/intro.png" alt="" width="2400" height="600" style="margin-bottom: 16px;" />
+
+Since the first v6 stable release, we have continuously rolled out new major features.
+Now that we reached [MUI X v6.10.x](https://github.com/mui/mui-x/releases), it's time to delve into the most recent additions to MUI X v6, and why you should get the latest version now.
+
+## Table of contents
+
+- [Table of contents](#table-of-contents)
+- [Date Pickers](#date-pickers)
+  - [Support for time zones](#support-for-time-zones)
+  - [Digital clock](#digital-clock)
+- [Data Grid](#data-grid)
+  - [Copy and Paste](#copy-and-paste)
+  - [Filter on column headers](#filter-on-column-headers)
+  - [Performance Enhancements: Smoother Scrolling and Expanding Rows](#performance-enhancements-smoother-scrolling-and-expanding-rows)
+- [Charts - alpha version 🎁](#charts---alpha-version-)
+- [What's next](#whats-next)
+- [Feedback](#feedback)
+
+## Date Pickers
+
+### Support for time zones
+
+No more hassle displaying dates on the time zones of your choice. 🌎
+
+You can effortlessly display and select dates and times in different time zones, without worrying about the conversion logic and keeping the stored value consistent with your backend.
+
+<video preload="metadata" style="margin-bottom: 16px;" autoplay muted loop controls>
+  <source src="/static/blog/mui-x-mid-v6-features/timezone.mp4" type="video/mp4">
+</video>
+
+Use adapters to control the original value and set the `timezone` prop to define in which timezone the Date, Time, or DateTime should be displayed.
```

Comment:
Fully agree, maybe it's worth saying that we are just using the date library time zone capability, but the adapters layer is clearly not the one to highlight here, it's an implementation detail.

Reply:
I removed the adapter as a whole and focused on the timezone prop. I feel like thats better to understand. If readers have more question, it will prompt them to check out the doc.